src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/EditingPreferencesPane.java

```diff
@@ -64,7 +64,14 @@ public EditingPreferencesPane(UIPrefs prefs,
             "Continue comment when inserting new line",
             prefs_.continueCommentsOnNewline(),
             "When enabled, pressing enter will continue comments on new lines. Press Shift + Enter to exit a comment."));
-      editingPanel.add(checkboxPref("Enable vim editing mode", prefs_.useVimMode()));
+      
+      vim_ = checkboxPref("Enable Vim editing mode", prefs_.useVimMode());
```

Comment:
Are these mutually exclusive? (i.e. should they be shown in a combo box rather than as checkboxes)